R/si_to_od.R

```diff
@@ -27,29 +27,37 @@
 #' odsf = si_to_od(origins, destinations)
 #' nrow(odsf) # no intrazonal flows
 #' plot(odsf)
-si_to_od = function(origins, destinations, max_dist = Inf) {
-  if(identical(origins, destinations)) {
+si_to_od = function(origins, destinations, max_dist = Inf, intrazonal = TRUE) {
+  if(identical(origins$geometry, destinations$geometry)) {
```

Comment:
This probably should be `sf::st_geometry(origins), sf::st_geometry(destinations)` as the column with geometries can have different names than `geometry`.

Reply:
Hello Nowosad,

Oh yes! That's very plausible. Let me correct that.

Thank you.

Reply:
I don't think this change is needed. And it's not related to #16